string camera
---
geometry_msgs/PointStamped[] pts3d
float64[] params
string[] param_names
sensor_msgs/Image image_annotated_left
sensor_msgs/Image image_annotated_right
